firmware/esp32_ttgo/client/LoRaClient.cpp

```diff
@@ -0,0 +1,40 @@
+#include "LoRaClient.h"
+
+bool LoRaClient::init()
+{
+    LL2.init();             // initialize Layer2
+    LL2.setInterval(10000); // set to zero to disable routing packets
+    return true;
+}
+
+void LoRaClient::loop()
+{
+    LL2.daemon();
+    struct Packet packet = LL2.popFromChatBuffer();
```

Comment:
This will need to be updated to `LL2.popFromWSBuffer()` once LL2 is updated to latest. It may need to be updated again, once we settle on a new name for that buffer based on discussion in https://github.com/sudomesh/LoRaLayer2/issues/3